charts/ceph-csi-rbd/values.yaml

```diff
@@ -113,6 +113,8 @@ provisioner:
   # Soft limit for maximum number of nested volume clones that are taken before
   # a flatten occurs
   softMaxCloneDepth: 4
+  # Maximum number of snapshots allowed on rbd image without flattening
+  maxSnapshotsOnImage: 450
```

Comment:
maybe mention that Ceph limits this to 510 - setting it higher will cause issues?

Reply:
Yes this will be documented when we add helm documentation, setting higher is not allowed as krbd will have issues.